src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -307,25 +307,29 @@ public function updateStatusAction(int $orderId, Request $request): RedirectResp
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            try {
-                $this->getCommandBus()->handle(
-                    new UpdateOrderStatusCommand(
```

Comment:
Nice we can reuse it UpdateOrderStatusCommand

I checked, it seems in UpdateOrderStatusCommand handler we dont check that the `$orderStatusId` is a valid ID, should we add the check here and in the handler ? Or does it happen at ObjectModel level ?